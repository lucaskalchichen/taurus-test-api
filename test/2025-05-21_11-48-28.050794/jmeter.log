2025-05-21 11:48:35,935 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:48:35,945 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:48:35,945 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:48:35,945 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794/jmeter-bzt.properties
2025-05-21 11:48:35,947 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794/system.properties
2025-05-21 11:48:35,950 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:48:35,950 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:48:35,950 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:48:35,951 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:48:35,954 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:48:35,954 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:48:35,954 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:48:35,954 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:48:35,954 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:48:35,960 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:48:35,961 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794
2025-05-21 11:48:35,961 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794
2025-05-21 11:48:35,962 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794
2025-05-21 11:48:35,969 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:48:35,969 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794'
2025-05-21 11:48:36,041 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:48:36,051 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:48:36,053 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:48:36,054 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794/modified_requests.jmx
2025-05-21 11:48:36,071 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:48:36,072 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:48:36,081 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:48:36,081 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:48:36,081 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:48:36,081 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:48:36,081 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:48:36,082 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:48:36,107 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:48:36,108 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:48:36,108 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:48:36,115 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:48:36,116 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:48:36,289 INFO o.a.j.JMeter: Running test (1747838916288)
2025-05-21 11:48:36,297 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:48:36,297 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:48:36,297 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:48:36,297 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:48:36,303 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:48:36,316 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:48:36,316 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:48:36,317 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:48:36,319 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:48:36,320 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:48:36,320 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:48:36,320 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:48:36,320 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:48:36,372 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:48:36,442 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:48:36,514 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:48:36,584 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:48:36,655 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:48:36,725 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:48:36,795 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:48:36,867 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:48:36,938 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:48:37,009 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:48:37,080 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:48:37,151 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:48:37,223 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:48:37,293 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:48:37,364 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:48:37,435 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:48:37,506 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:48:37,577 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:48:37,647 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:48:37,719 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:48:37,790 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:48:37,861 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:48:37,931 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:48:38,003 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:48:38,073 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:48:38,145 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:48:38,216 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:48:38,286 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:48:38,357 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:48:38,428 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:48:38,499 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:48:38,523 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:48:38,523 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:48:38,528 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:48:38,528 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:48:38,530 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:48:38,530 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:48:38,534 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 11:48:38,534 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:48:38,535 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 11:48:38,535 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:48:38,535 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:48:38,535 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:48:38,536 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:48:38,536 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:48:38,537 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:48:38,537 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:48:38,537 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:48:38,537 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:48:38,538 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 11:48:38,538 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:48:38,539 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:48:38,539 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:48:38,540 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:48:38,540 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:48:38,540 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:48:38,540 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:48:38,541 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:48:38,541 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:48:38,542 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:48:38,542 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:48:38,542 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:48:38,542 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:48:38,543 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:48:38,543 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:48:38,544 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:48:38,544 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:48:38,544 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:48:38,544 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:48:38,545 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:48:38,545 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:48:38,546 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 11:48:38,546 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:48:38,546 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:48:38,546 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 11:48:38,547 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:48:38,548 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 11:48:38,548 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:48:38,549 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:48:38,550 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:48:38,550 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 11:48:38,550 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:48:38,551 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 11:48:38,551 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:48:38,570 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:48:38,574 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 11:48:38,574 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:48:38,642 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:48:38,648 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 11:48:38,648 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:48:38,713 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:48:38,721 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 11:48:38,721 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:48:38,784 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:48:38,801 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 11:48:38,802 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:48:38,855 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:48:38,864 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 11:48:38,864 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:48:38,925 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:48:38,931 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 11:48:38,931 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:48:38,996 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:48:39,004 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 11:48:39,004 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:48:39,068 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:48:39,076 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 11:48:39,076 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:48:39,139 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:48:39,146 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 11:48:39,146 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:48:39,210 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:48:39,217 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 11:48:39,217 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:48:39,217 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:48:39,218 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
