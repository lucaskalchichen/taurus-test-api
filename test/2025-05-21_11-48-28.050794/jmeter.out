Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-48-28.050794/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:48:36 ART (1747838916101)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 11:48:39 ART (1747838919218)
... end of run
